src/c++/perf_analyzer/inference_profiler.h

```diff
@@ -234,6 +234,11 @@ class InferenceProfiler {
         current_value += step;
       } while (((current_value <= end) || (end == static_cast<T>(NO_LIMIT))) &&
                (meets_threshold));
+      // If there was only one concurrency we swept over and it did not meet the
+      // stability threshold, we should return an error.
+      if (current_value == (start + step) && meets_threshold == false) {
+        return cb::Error(" ");
```

Comment:
What is the behavior here? Does it still print out "Failed to obtain stable measurement"?  I still want that print but return non-zero.

Note that this is easy to reproduce if you just pass in `-s 0` to a perf_analyzer command, as it will never stabilize.

Reply:
Still prints that message. And now PA returns non-zero exit code when this happens 👍